dev/Deployment/DeploymentManager.cpp

```diff
@@ -48,6 +48,7 @@ inline void Initialize_StopSuccessActivity(
 
 namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implementation
 {
+    static bool isInitializedByDefault{ false };
```

Comment:
can we rename for slight clarity?
```suggestion
    static bool isAutoInitialized{ false };
```

Reply:
Changed name throughout file in latest commit. Thanks!